README.md

```diff
@@ -1 +1,93 @@
-# downtown-event
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name : World Energy Event ♻️
+
+This is the copy for the World Energy Event project based on the original idea from Cindy Shin
+
+# Screenshots
+- Desktop Layout
+
+![Desktop_Layout](assets/images/desktop.png)
+
+- Mobile Layout
+
+![Mobile_Layout](assets/images/mobile.png)
+
+- The into with loom video link
+
+https://www.loom.com/share/2c574b34a32647248a49e7cc5edfb589
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML 
+- CSS
+- Javascript
+- VScode
+- Git & GitHub
+## Live Demo link
+
+[Live Demo Link](https://eckybrazzz.github.io/World-Energy-Event/)
```

Comment:
**Suggested Changes**
- [ ] The video link opens a blank page. Let us please try and fix this so that our website is accessible from anywhere.

Reply:
This link only works if the PR is merged

[Live Demo Link](https://eckybrazzz.github.io/World-Energy-Event/)

Reply:
Fix with gh-pages, selecting the current working branch.